plugins/services/src/js/service-configuration/PodGeneralConfigSection.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import ConfigurationMapKeyValueRow from '../components/ConfigurationMapKeyValueRow';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import Units from '../../../../../src/js/utils/Units';
+
+function getContainerResourceSummary(resource, appConfig) {
+  let summary = (appConfig.containers || []).reduce((memo, container) => {
+    if (container.resources && container.resources[resource]) {
+      let value = container.resources[resource];
+      memo.value += value;
+      memo.parts.push(
+        `${Units.formatResource(resource, value)} ${container.name}`
+      );
+    }
+    memo.value += (container.resources || {})[resource] || 0;
+    return memo;
+  }, {value: 0, parts: []});
+
+  if (!summary.value) {
+    return <em>Not Supported</em>;
+  }
+
+  return `${summary.value} (${summary.parts.join(', ')})`;
+}
+
+module.exports = ({appConfig}) => {
+  return (
+    <div>
+      <ConfigurationMapHeading level={1}>General</ConfigurationMapHeading>
+      <ConfigurationMapSection key="pod-general-section">
+        <ConfigurationMapKeyValueRow
+          label="Service ID"
+          value={appConfig.id} />
+
+        <ConfigurationMapKeyValueRow
+          label="Instances"
+          value={() => {
+            if (appConfig.scaling && appConfig.scaling.kind === 'fixed') {
+              let expr = `${appConfig.scaling.instances}`;
+              if (appConfig.scaling.maxInstances) {
+                expr += ` (Max ${appConfig.scaling.maxInstances})`;
+              }
+
+              return expr;
+            }
+
+            return null;
+          }} />
+
+        <ConfigurationMapKeyValueRow
+          label="CPU"
+          value={getContainerResourceSummary.bind({}, 'cpus', appConfig)} />
+        <ConfigurationMapKeyValueRow
+          label="Memory"
+          value={getContainerResourceSummary.bind({}, 'mem', appConfig)} />
+        <ConfigurationMapKeyValueRow
+          label="Disk"
+          value={getContainerResourceSummary.bind({}, 'disk', appConfig)} />
+        <ConfigurationMapKeyValueRow
+          label="GPU"
+          value={getContainerResourceSummary.bind({}, 'gpu', appConfig)} />
+
+        <ConfigurationMapKeyValueRow
+          label="Backoff"
+          value={() => {
+            return appConfig.scheduling &&
```

Comment:
you can use: `value={findNestedPropertyInObject(appConfig, 'scheduling.backoff.backoff')}` for these